modules/peridynamics/src/mesh/PeridynamicsMesh.C

```diff
@@ -201,6 +201,8 @@ PeridynamicsMesh::createPeridynamicsMeshData(
     _pdnode_coord[id] = fe_elem->centroid();
     _pdnode_avg_spacing[id] = dist_sum / n_fe_neighbors;
     _pdnode_horiz_rad[id] = (_has_horiz_num ? _horiz_num * dist_sum / n_fe_neighbors : _horiz_rad);
+    // this needs to be fixed as well. MooseMesh does not have Assembly so
+    // that makes this harder here
```

Comment:
This needs to be fixed.

Reply:
@bwspenc will know more about this, but IIRC this peridynamics thing has no concept for Cartesian, RZ, RSpherical. So, this may stay the way it is...

Reply:
Peridynamics is only implemented for the Cartesian case.  We probably need to add an error if you try to use it with any other coord type.